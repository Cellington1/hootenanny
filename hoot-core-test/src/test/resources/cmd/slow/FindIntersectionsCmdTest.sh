#!/bin/bash
set -e

mkdir -p $HOOT_HOME/test-output/cmd/slow/FindIntersectionsCmdTest

inputfile=$HOOT_HOME/test-files/DcGisRoads.osm
comparefile=$HOOT_HOME/hoot-core-test/src/test/resources/cmd/slow/FindIntersectionsCmdTest/out.osm
outputfile=$HOOT_HOME/test-output/cmd/slow/FindIntersectionsCmdTest/out.osm

# Run the command. Note that we sanitize output to remove timestamps
hoot find-intersections $inputfile $outputfile  |\
                        sed "s/^[0-9]\{2\}:[0-9]\{2\}:[0-9]\{2\}\.[0-9]\{3\} //g"

# Checking output
hoot is-match $comparefile $outputfile || diff $comparefile $outputfile
