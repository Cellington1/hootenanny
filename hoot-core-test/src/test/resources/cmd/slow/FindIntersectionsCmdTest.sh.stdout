INFO  ...re/ops/FindIntersectionsOp.cpp(  66) 0 Relations found.
INFO  ...re/ops/FindIntersectionsOp.cpp(  69) 0 Relations found, after removal
Reprojecting 0 / 4324         Reprojecting 1000 / 4324         Reprojecting 2000 / 4324         Reprojecting 3000 / 4324         Reprojecting 4000 / 4324         
INFO  ...nflate/DuplicateWayRemover.cpp(  64) Removing duplicate ways...
INFO  ...late/SuperfluousWayRemover.cpp(  59) Removing superfluous ways...
INFO  ...itter/IntersectionSplitter.cpp( 109) Splitting intersections...
  Intersection splitter todo: 0       
INFO  ...re/ops/FindIntersectionsOp.cpp(  84) Assuming drives on right.
INFO  ...re/ops/FindIntersectionsOp.cpp( 102) 224 Intersections found.
INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  56) Applying operation: hoot::ReprojectToPlanarOp
INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  56) Applying operation: hoot::DuplicateWayRemover
INFO  ...nflate/DuplicateWayRemover.cpp(  64) Removing duplicate ways...
INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  56) Applying operation: hoot::SuperfluousWayRemover
INFO  ...late/SuperfluousWayRemover.cpp(  59) Removing superfluous ways...
INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  56) Applying operation: hoot::IntersectionSplitter
INFO  ...itter/IntersectionSplitter.cpp( 109) Splitting intersections...

INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  56) Applying operation: hoot::UnlikelyIntersectionRemover
INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  56) Applying operation: hoot::DualWaySplitter
INFO  ...e/splitter/DualWaySplitter.cpp(  83) Assuming drives on right.
INFO  ...e/splitter/DualWaySplitter.cpp( 246) Splitting divided ways...

INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  56) Applying operation: hoot::ImpliedDividedMarker
INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  56) Applying operation: hoot::DuplicateNameRemover
INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  56) Applying operation: hoot::SmallWayMerger
INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  74) Applying visitor: hoot::RemoveEmptyAreasVisitor
INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  74) Applying visitor: hoot::RemoveDuplicateAreaVisitor
INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  56) Applying operation: hoot::NoInformationElementRemover
Reprojecting 0 / 206         
